buildconfig/Jenkins/Conda/publish-to-anaconda

```diff
@@ -39,9 +40,9 @@ conda activate $CONDA_ENV_NAME
 for package in "$@"
 do
     # Skip the Workspace, Token, and Channel args, then upload every other package to the given User
-    if [ $package != $1 ] && [ $package != $2 ] && [ $package != $3 ] && [[ $package == *"mantid"* ]]
+    if [ $package != $1 ] && [ $package != $2 ] && [ $package != $3 ] && [ $package != $4 ] && [[ $package == *"mantid"* ]]
```

Comment:
I'm sure there must be a better way of doing this, but can't think what at the moment...

Reply:
Would it make sense to capture the first 4 arguments at the top alongside `WORKSPACE` and then `shift 4` to swallow those arguments leaving the packages?

Reply:
$@ captures things that have been "shifted" past, so it wouldn't work here.

Reply:
Oh. That's not my experience:

```
$ cat test.sh
echo $1
echo $2
echo $3


shift 2
echo

echo $@

$ bash test.sh 1 2 3
1
2
3

3
```